README.md

```diff
@@ -109,6 +109,12 @@ To increase the size of the virtual hard drive, i. e. times 20 (see [virtualisat
 
 Notice that for this option to become effective you may also need to delete previous block device files created by qemu (`nixos.qcow2`).
 
+Also note, that in some cases you might want to use nixos-shell as a remote builder provider, to enable storing changes to the nix store in the block device rather than in tmpfs:
```

Comment:
```suggestion
Notice that changes in the nix store are written to an overlayfs backed by tmpfs rather than the block device
that is configured by `virtualisation.diskSize`. This tmpfs can be changed however by using:
```